app/javascript/components/public_health/actions/CloseRecords.js

```diff
@@ -76,7 +79,8 @@ class CloseRecords extends React.Component {
           </Form.Group>
           <Form.Group controlId="reasoning">
             <Form.Label>Please include any additional details:</Form.Label>
-            <Form.Control as="textarea" rows="2" onChange={this.handleChange} />
+            <Form.Control as="textarea" rows="2" maxLength={MAX_NOTES_LENGTH} onChange={this.handleChange} />
+            <Form.Label className="notes-character-limit"> {MAX_NOTES_LENGTH - this.state.reasoning.length} characters remaining </Form.Label>
```

Comment:
should be a div with class `character-limit-text`

Reply:
Thanks! I fixed it in the other file, but forgot about this one.

Reply:
this one still needs to be fixed FYI